src/mmw/js/src/data_catalog/templates/resultDetails.html

```diff
@@ -0,0 +1,35 @@
+<div class="result">
+    <div class="result-detail-header">
+        <h2>
+            {{ title }}
+        </h2>
+        <button type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+        </button>
+        <a href="#"><i class="fa fa-search-plus"></i> Zoom to extent</a>
+    </div>
+    <hr>
+    {% if author %}
+        <p>
+            <i class="fa fa-user"></i>{{ author }}
+        </p>
+    {% endif %}
+    {% if activeCatalog != 'cuahsi' %}
```

Comment:
I tried something similar, but there is a bit of an overlap in fields between the two catalogs, so I wanted to be more explicit. We may end up having to change this in the future too depending on how much the detail panes diverge between the catalogs.